projects/1.16/assets/zombie-extreme/zombie-extreme/lang/zh_cn.json

```
@@ -0,0 +1,408 @@
+{
+  "advancements.alternateworldadvancement.descr": "核战，人类的终末……",
+  "advancements.alternateworldadvancement.title": "死海",
+  "advancements.anddoesnotbreakwithoutwork.descr": "制作钛镐",
+  "advancements.anddoesnotbreakwithoutwork.title": "获得钛镐",
+  "advancements.darkpearl.descr": "熔炼一个暗金属锭",
+  "advancements.darkpearl.title": "暗珍珠",
+  "advancements.fleshofthe_infected.descr": "寻找远古血肉",
+  "advancements.rubies.descr": "挖红宝石",
+  "advancements.rubies.title": "红宝石！",
+  "advancements.titanignot.descr": "熔炼钛锭",
+  "advancements.titanignot.title": "获得钛锭",
+  "advancements.titanpower.descr": "权力在手",
+  "advancements.titanpower.title": "钛之力！",
+  "advancements.whatam_idoing.descr": "合成血肉锭",
+  "advancements.whatam_idoing.title": "我在做什么......？",
```

```suggestion
  "advancements.whatam_idoing.title": "我在做什么……？",
```
中文输入法下`Shift`+`6`可打出省略号（`……`），请全文通查